### PR TITLE
Add Javascript logic 

### PR DESCRIPTION
**Pull request by bwnstck**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,12 +1,59 @@
 import "./style.css";
+import { createElement } from "./utils/elements";
+
+var startInterval;
```
Comment (lmachens) [inline src/index.js:4]:
```suggestion
let startInterval= null;
```
`null` indicates that you will set a value soon.
https://javascript.info/types#the-null-value

File: src/index.js [hunk 2]
```diff
@@ -1,12 +1,59 @@
...
+    className: "input--time",
+    placeholder: "hh:mm:ss",
+  });
+
+  const timerButtonStart = createElement("button", {
+    className: "button--startCount",
+    innerText: "Count Down",
+    onclick: () => {
+      startTimer();
+    },
```
Comment (lmachens) [inline src/index.js:40]: `startTimer` is already a function, thus, you don't need to create a new one.
```suggestion
    onclick: startTimer,
```

**Review by lmachens — COMMENTED** [review]
gj

File: src/index.js (hunk 1 above)
Comment (bwnstck) [reply src/index.js:4]: MDN has an example using var… is it just old or do they use it with purpose?
https://developer.mozilla.org/de/docs/Web/API/WindowOrWorkerGlobalScope/setInterval#Examples

**Pull request merged**